app/frontend/src/components/Navigation/Navigation.tsx

```diff
@@ -10,8 +10,10 @@ import {
 import classNames from "classnames";
 import { CloseIcon, MenuIcon } from "components/Icons";
 import ExternalNavButton from "components/Navigation/ExternalNavButton";
+import NotificationBadge from "components/NotificationBadge";
 import { useAuthContext } from "features/auth/AuthProvider";
 import useAuthStyles from "features/auth/useAuthStyles";
+import useNotifications from "features/useNotifications";
```

Comment:
I think you forgot to add these to git!

Reply:
You mean those lines wasn't added to the commit ? @aapeliv

Reply:
Sorry I'm mistaken here, please ignore the comment! I didn't realize these files were already in the tree, I though they were files you added.